ale_linters/erlang/dialyzer.vim

```diff
@@ -47,14 +49,20 @@ function! ale_linters#erlang#dialyzer#GetExecutable(buffer) abort
 endfunction
 
 function! ale_linters#erlang#dialyzer#GetCommand(buffer) abort
+    let l:options = ale#Var(a:buffer, 'erlang_dialyzer_options')
+
+    if len(l:options) == 0
+        let l:options = '-Wunmatched_returns'
```

Comment:
I am not one of the main reviewer, but wouldn't it be possible to put this inside the default of the linter variables and remove the `if` altotogether? For example:

```vimscript
let g:ale_erlang_dialyzer_options =
\   get(g:, 'ale_erlang_dialyzer_options',
\       '-Wunmatched_returns -Werror_handling -Wrace_conditions -Wunderspecs')
```

Reply:
You're totally right, I can't believe I missed this ;)

Just fixed it.